weather_app/app.py

```diff
@@ -12,22 +12,28 @@ def parser():
     return args
 
 
-args = parser()
-
-
-def coordinates(city):
+def get_city_coordinates(city):
     r = requests.get(f'https://www.geocode.farm/v3/json/forward/?addr={city}&lang=en')
     lat_lon = r.json()
-    return lat_lon['geocoding_results']['RESULTS'][0]['COORDINATES']
+    try:
+        return lat_lon['geocoding_results']['RESULTS'][0]['COORDINATES']
+    except KeyError:
+        print(lat_lon['geocoding_results']['STATUS'])
 
 
 def fetch_weather(city):
-    city_lat = float(coordinates(city)['latitude'])
-    city_lon = float(coordinates(city)['longitude'])
-    r = requests.get(f'https://fcc-weather-api.glitch.me/api/current?lat={city_lat}&lon={city_lon}')
-    forcast = r.json()
-    return [round(forcast['main']['temp']), round(forcast['main']['feels_like']), forcast['weather'][0]['main']]
-
-
-output = fetch_weather(city=args.city)
-print(f"Current temp: {output[0]} °C, Feel-like temp: {output[1]} °C, Sky: {output[2]}")
+    try:
+        city_lat = float(get_city_coordinates(city)['latitude'])
+        city_lon = float(get_city_coordinates(city)['longitude'])
+        r = requests.get(f'https://fcc-weather-api.glitch.me/api/current?lat={city_lat}&lon={city_lon}')
+        forcast = r.json()
+        return [round(forcast['main']['temp']), round(forcast['main']['feels_like']), forcast['weather'][0]['main']]
```

Comment:
Let's discuss review hint https://github.com/I-Behdar/Weather_Forcast/issues/1#issuecomment-1383210756 on the call today